### PR TITLE
Update to hypha rpc

### PR DESCRIPTION
**Pull request by konradoo11**
Updated to the latest Hypha RPC.

@oeway 
**Current issue is:** 
In the Log Section when running the app: Service not found error when trying to acquire the services.

MicroscopeControl_Console
MicroscopeControl_LOG-section

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -0,0 +1,1102 @@
...
+  const [similarityService, setSimilarityService] = useState(null);
+  const appendLog = (message) => {
+      setLog(prevLog => prevLog + message + '\n');
+  };
+
+  useEffect(() => {
+    const initializeWebRPC = async () => {
+        try {
+            appendLog('Connecting to server...');
+            const server = await hyphaWebsocketClient.connectToServer({"server_url": "https://hypha.aicell.io", "webrtc": true})
```
Comment (konradoo11) [inline src/main.jsx:83]: @oeway Is this correct?

File: .DS_Store
Comment (oeway) [inline .DS_Store:1]: Please remove this file and add it to git ignore.

**Review by oeway — COMMENTED** [review]
Is it intended to remove all the python scripts?

File: src/main.jsx (hunk 1 above)
Comment (oeway) [reply src/main.jsx:83]: The issue is that you connect to the server as anonymous user (from user's own workspace), and when you call `getSerivce("interactive-segmentation")`, the relative service id "interactive-segmentation" assumes you have that service registered under the anonymous user's own workspace. This is why it gives you service not found. You should use the full service id in the format of "{workspace}/interactive-segmentation", you need to get the actual workspace id where you registered `interactive-segmentation` to replace `{workspace}`.

Another note, please implement a login feature following:
https://docs.amun.ai/#/getting-started?id=user-login-and-token-based-authentication
We don't want anyone has access to the service eventually.

**Comment by konradoo11** [comment]
> Is it intended to remove all the python scripts?

No, I recently realised that they could remain as they were before. I will fix that.

**Pull request closed without merging**